src/test/java/eu/dissco/orchestration/backend/service/MachineAnnotationServiceServiceTest.java

```diff
@@ -76,22 +98,80 @@ void testCreateMas() throws Exception {
     var expected = givenMasSingleJsonApiWrapper();
     var mas = givenMas();
     given(handleService.createNewHandle(HandleType.MACHINE_ANNOTATION_SERVICE)).willReturn(HANDLE);
+    given(properties.getKafkaHost()).willReturn("kafka.svc.cluster.local:9092");
```

Comment:
It looks like you declare this given in every test. Maybe put it in the setup instead?